src/app/search/directives/search-story.component.css

```diff
@@ -0,0 +1,121 @@
+.story-image {
+  position: relative;
+  float: left;
+  width: 120px;
+  height: 120px;
+  margin-bottom: 5px;
+}
+image-loader {
+  width: 100%;
+  height: 100%;
+  display: block;
+}
+
+.status {
+  position: absolute;
+  top: 0;
+  right: 0;
+  padding: 3px 10px;
+  z-index: 10;
+  color: #fff;
+  background-color: #444;
+  font-weight: 400;
+  font-size: 11px;
+  text-transform: uppercase;
+  line-height: 18px;
+}
+.status.unpublished {
+  background-color: #ee4d3e;
+}
+
+.story-detail {
+  position: relative;
+  float: right;
+  padding: 10px;
+  width: 200px;
+  height: 120px;
+  line-height: 20px;
+}
+.story-title {
+  font-size: 16px;
+  max-height: 40px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+.series-title {
+  max-height: 40px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.meta {
+  position: absolute;
+  bottom: 0;
+  width: 180px;
+  line-height: 20px;
+}
+.duration, .modified, .audio-total {
+  width: calc(100%/3);
+}
+.modified {
+  text-align: right;
+}
+.audio-total {
+}
+.audio-total > i {
+  /* TODO: this isn't the correct icon */
+  display: inline-block;
+  transform: rotate(90deg);
+  margin-right: -5px;
+}
+
+.story-tags {
+  clear: left;
+  display: flex;
+  justify-content: flex-start;
+  flex-flow: row wrap;
+  width: 320px;
+  padding: 0 10px;
+  max-height: 20px;
+  line-height: 20px;
+  overflow: hidden;
+}
+.story-tags > span {
+  background-color: #f0f0f0;
+  padding: 0 4px;
+}
+
+.story-tags > span + span {
+  margin-left: 4px;
+}
+
+.story-description {
+  clear: left;
+  width: 320px;
+  padding: 0 10px;
+  margin-top: 5px;
+  margin-bottom: 10px;
+}
+
+.three-lines {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  max-height: 60px;
+  line-height: 20px;
+  -webkit-line-clamp: 3;
+  display: block;
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
```

Comment:
This will only truncate with ellipsis in webkit browsers. The fallback is truncation without ellipsis. The mockup actually has a Read More link. I wasn't sure what the intention was there, so I didn't add it yet. Cross browser multiline ellipsis looks kind of gross http://codepen.io/romanrudenko/pen/ymHFh, so I'm gonna leave this alone for now until figuring out what Read More or whatever this truncation should actually do.
<img width="1143" alt="screen shot 2016-09-23 at 1 19 25 pm" src="https://cloud.githubusercontent.com/assets/2250413/18796806/8894974c-8190-11e6-8d8e-3703fae35109.png">